### DISCUSSION TITLE
Manim v0.12.0

### DISCUSSION THREAD

**Discussion by behackl** [ManimCommunity/manim#2252] [Announcements]
The full changelog can be found at https://docs.manim.community/en/latest/changelog/0.12.0-changelog.html.

<hr /><em>This discussion was created from the release <a href='https://github.com/ManimCommunity/manim/releases/tag/v0.12.0'>Manim v0.12.0</a>.</em>

**Comment by hackyfemmyboi** [comment]
I've been trying to get manim. It says I have it, but when i type Manim into the CMD, it says manim is not recognized.

**Reply by behackl** [reply]
There is very little that you give us to work with. How did you try to install manim? You speak of a CMD, so I suppose this is Windows; did you open a new terminal after installing? Did you use chocolatey?

More info please.

**Reply by Darylgolden** [reply]
Have you read our [troubleshooting guide](https://docs.manim.community/en/stable/installation/troubleshooting.html)?

**Reply by hackyfemmyboi** [reply]
Yes. I'll send the picture of the problem
![image](https://user-images.githubusercontent.com/93700797/140467392-6c5e1c27-a24c-4fd0-b400-e6101d902c86.png)
![image](https://user-images.githubusercontent.com/93700797/140467466-d59566cd-b552-496b-9301-5e455f510545.png)
![image](https://user-images.githubusercontent.com/93700797/140467476-e23ad7c6-9b08-485f-8583-09d691869c44.png)

**Reply by Darylgolden** [reply]
You are supposed to pip install `manim`, not `manimce`. Do `pip uninstall manimce` and `pip install manim`. By any chance did you read an old version of the documentation?

**Reply by hackyfemmyboi** [reply]
![image](https://user-images.githubusercontent.com/93700797/140467803-e22b8f86-70dd-4bdc-bb7c-ed54781c4a8a.png)
shiiiiiiiiizzz

**Reply by Darylgolden** [reply]
Do you own the computer you're trying to install manim on? Or do you not have administrator permissions?

**Reply by hackyfemmyboi** [reply]
i dont have admin
is there a wrkaround?

**Reply by hackyfemmyboi** [reply]
also,
![image](https://user-images.githubusercontent.com/93700797/140468087-51d756a1-9e2a-42aa-9e89-5a5194e17323.png)

**Reply by Darylgolden** [reply]
Try `pip uninstall manimce --user` and `pip install manim --user`.

**Reply by hackyfemmyboi** [reply]
doesnt work sorry

**Reply by hackyfemmyboi** [reply]
![Uploading image.png…]()

**Reply by hackyfemmyboi** [reply]
i was also wondering if it's possible to animate 3B1B's pi creatures

**Reply by Darylgolden** [reply]
You need to provide more details for us to effectively help you. Can you format the full terminal output as a code block, rather than sending screenshots?

**Reply by hackyfemmyboi** [reply]
wait, how do i do that here?

**Reply by hackyfemmyboi** [reply]
ah

**Reply by hackyfemmyboi** [reply]
`Microsoft Windows [Version 10.0.18363.418]
(c) 2019 Microsoft Corporation. All rights reserved.

C:\Users\Rene>manim
'manim' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Rene>pip list
Package            Version
------------------ ---------
backcall           0.2.0
cairocffi          1.2.0
certifi            2021.10.8
cffi               1.14.6
charset-normalizer 2.0.7
click              8.0.3
colorama           0.4.4
colour             0.1.5
commonmark         0.9.1
cycler             0.10.0
ffmpeg             1.4
glcontext          2.3.4
idna               3.3
isosurfaces        0.1.0
jedi               0.18.0
kiwisolver         1.3.2
matplotlib-inline  0.1.3
moderngl           5.6.4
mpmath             1.2.1
multipledispatch   0.6.0
numpy              1.21.3
odeintw            0.1.1
opencv-python      4.5.3.56
pangocairocffi     0.4.0
pangocffi          0.8.0
parso              0.8.2
pickleshare        0.7.5
Pillow             8.4.0
pip                21.2.3
progressbar        2.5
prompt-toolkit     3.0.20
pycairo            1.20.1
pycparser          2.20
pydub              0.25.1
pyglet             1.5.21
Pygments           2.10.0
pyparsing          2.4.7
pyreadline         2.1
pyrr               0.10.3
python-dateutil    2.8.2
rich               6.2.0
scipy              1.7.1
setuptools         57.4.0
six                1.16.0
skia-pathops       0.7.0
sympy              1.9
tqdm               4.62.3
traitlets          5.1.0
typing-extensions  3.10.0.2
urllib3            1.26.7
validators         0.18.2
watchdog           2.1.6
wcwidth            0.2.5
wheel              0.37.0

C:\Users\Rene>pip install manimce
Collecting manimce
  Using cached manimce-0.1.1.post2-py3-none-any.whl (249 kB)
Requirement already satisfied: tqdm in c:\python39\lib\site-packages (from manimce) (4.62.3)
Requirement already satisfied: rich<7.0,>=6.0 in c:\python39\lib\site-packages (from manimce) (6.2.0)
Requirement already satisfied: pydub in c:\python39\lib\site-packages (from manimce) (0.25.1)
Requirement already satisfied: Pillow in c:\python39\lib\site-packages (from manimce) (8.4.0)
Requirement already satisfied: progressbar in c:\python39\lib\site-packages (from manimce) (2.5)
Requirement already satisfied: scipy in c:\python39\lib\site-packages (from manimce) (1.7.1)
Requirement already satisfied: colour in c:\python39\lib\site-packages (from manimce) (0.1.5)
Requirement already satisfied: cairocffi<2.0.0,>=1.1.0 in c:\python39\lib\site-packages (from manimce) (1.2.0)
Requirement already satisfied: numpy in c:\python39\lib\site-packages (from manimce) (1.21.3)
Requirement already satisfied: pangocffi<0.9.0,>=0.8.0 in c:\python39\lib\site-packages (from manimce) (0.8.0)
Requirement already satisfied: pygments in c:\python39\lib\site-packages (from manimce) (2.10.0)
Requirement already satisfied: pangocairocffi<0.5.0,>=0.4.0 in c:\python39\lib\site-packages (from manimce) (0.4.0)
Requirement already satisfied: pycairo<2.0,>=1.19 in c:\python39\lib\site-packages (from manimce) (1.20.1)
Requirement already satisfied: cffi>=1.1.0 in c:\python39\lib\site-packages (from cairocffi<2.0.0,>=1.1.0->manimce) (1.14.6)
Requirement already satisfied: pycparser in c:\python39\lib\site-packages (from cffi>=1.1.0->cairocffi<2.0.0,>=1.1.0->manimce) (2.20)
Requirement already satisfied: colorama<0.5.0,>=0.4.0 in c:\python39\lib\site-packages (from rich<7.0,>=6.0->manimce) (0.4.4)
Requirement already satisfied: typing-extensions<4.0.0,>=3.7.4 in c:\python39\lib\site-packages (from rich<7.0,>=6.0->manimce) (3.10.0.2)
Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in c:\python39\lib\site-packages (from rich<7.0,>=6.0->manimce) (0.9.1)
Installing collected packages: manimce
  WARNING: Failed to write executable - trying to use .deleteme logic
ERROR: Could not install packages due to an OSError: [WinError 2] The system cannot find the file specified: 'C:\\Python39\\Scripts\\manim.exe' -> 'C:\\Python39\\Scripts\\manim.exe.deleteme'


C:\Users\Rene>manim
'manim' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Rene>pip install manim
Collecting manim
  Using cached manim-0.12.0-py3-none-any.whl (505 kB)
Requirement already satisfied: tqdm in c:\python39\lib\site-packages (from manim) (4.62.3)
Collecting moderngl-window<3.0.0,>=2.3.0
  Using cached moderngl_window-2.4.0-py3-none-any.whl (362 kB)
Collecting requests
  Using cached requests-2.26.0-py2.py3-none-any.whl (62 kB)
Requirement already satisfied: pydub in c:\python39\lib\site-packages (from manim) (0.25.1)
Requirement already satisfied: numpy<2.0,>=1.9 in c:\python39\lib\site-packages (from manim) (1.21.3)
Collecting cloup<0.8.0,>=0.7.0
  Using cached cloup-0.7.1-py2.py3-none-any.whl (23 kB)
Collecting networkx<3.0,>=2.5
  Using cached networkx-2.6.3-py3-none-any.whl (1.9 MB)
Collecting decorator<6.0.0,>=5.0.7
  Using cached decorator-5.1.0-py3-none-any.whl (9.1 kB)
Collecting mapbox-earcut<0.13.0,>=0.12.10
  Downloading mapbox_earcut-0.12.11-cp39-cp39-win_amd64.whl (67 kB)
     |████████████████████████████████| 67 kB 4.5 MB/s
Requirement already satisfied: Pillow in c:\python39\lib\site-packages (from manim) (8.4.0)
Requirement already satisfied: isosurfaces==0.1.0 in c:\python39\lib\site-packages (from manim) (0.1.0)
Requirement already satisfied: colour in c:\python39\lib\site-packages (from manim) (0.1.5)
Collecting screeninfo<0.7.0,>=0.6.7
  Using cached screeninfo-0.6.7-py3-none-any.whl
Requirement already satisfied: setuptools in c:\python39\lib\site-packages (from manim) (57.4.0)
Requirement already satisfied: skia-pathops<0.8.0,>=0.7.0 in c:\python39\lib\site-packages (from manim) (0.7.0)
Collecting manimpango<0.4.0,>=0.3.0
  Using cached ManimPango-0.3.1-cp39-cp39-win_amd64.whl (3.9 MB)
Requirement already satisfied: scipy in c:\python39\lib\site-packages (from manim) (1.7.1)
Requirement already satisfied: watchdog in c:\python39\lib\site-packages (from manim) (2.1.6)
Requirement already satisfied: click>=7.1 in c:\python39\lib\site-packages (from manim) (8.0.3)
Requirement already satisfied: rich>=6.0 in c:\python39\lib\site-packages (from manim) (6.2.0)
Collecting click-default-group
  Using cached click_default_group-1.2.2-py3-none-any.whl
Requirement already satisfied: moderngl<6.0.0,>=5.6.3 in c:\python39\lib\site-packages (from manim) (5.6.4)
Requirement already satisfied: pygments in c:\python39\lib\site-packages (from manim) (2.10.0)
Requirement already satisfied: pycairo<2.0,>=1.19 in c:\python39\lib\site-packages (from manim) (1.20.1)
Requirement already satisfied: colorama in c:\python39\lib\site-packages (from click>=7.1->manim) (0.4.4)
Requirement already satisfied: glcontext<3,>=2 in c:\python39\lib\site-packages (from moderngl<6.0.0,>=5.6.3->manim) (2.3.4)
Requirement already satisfied: pyrr<1,>=0.10.3 in c:\python39\lib\site-packages (from moderngl-window<3.0.0,>=2.3.0->manim) (0.10.3)
Requirement already satisfied: pyglet<2,>=1.5.8 in c:\python39\lib\site-packages (from moderngl-window<3.0.0,>=2.3.0->manim) (1.5.21)
Requirement already satisfied: multipledispatch in c:\python39\lib\site-packages (from pyrr<1,>=0.10.3->moderngl-window<3.0.0,>=2.3.0->manim) (0.6.0)
Requirement already satisfied: typing-extensions<4.0.0,>=3.7.4 in c:\python39\lib\site-packages (from rich>=6.0->manim) (3.10.0.2)
Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in c:\python39\lib\site-packages (from rich>=6.0->manim) (0.9.1)
Requirement already satisfied: six in c:\python39\lib\site-packages (from multipledispatch->pyrr<1,>=0.10.3->moderngl-window<3.0.0,>=2.3.0->manim) (1.16.0)
Requirement already satisfied: charset-normalizer~=2.0.0 in c:\python39\lib\site-packages (from requests->manim) (2.0.7)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in c:\python39\lib\site-packages (from requests->manim) (1.26.7)
Requirement already satisfied: certifi>=2017.4.17 in c:\python39\lib\site-packages (from requests->manim) (2021.10.8)
Requirement already satisfied: idna<4,>=2.5 in c:\python39\lib\site-packages (from requests->manim) (3.3)
Installing collected packages: screeninfo, requests, networkx, moderngl-window, mapbox-earcut, manimpango, decorator, cloup, click-default-group, manim
ERROR: Could not install packages due to an OSError: [WinError 5] Access is denied: 'C:\\Python39\\share'
Consider using the `--user` option or check the permissions.


C:\Users\Rene>manim
'manim' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Rene>pip uninstall manimce
Found existing installation: manimce 0.1.1.post2
Uninstalling manimce-0.1.1.post2:
  Would remove:
    c:\python39\lib\site-packages\manim\*
    c:\python39\lib\site-packages\manimce-0.1.1.post2.dist-info\*
  Would not remove (might be manually added):
    c:\python39\lib\site-packages\manim\manimce-0.1.1.post2.dist-info\LICENSE
    c:\python39\lib\site-packages\manim\manimce-0.1.1.post2.dist-info\LICENSE.community
    c:\python39\lib\site-packages\manim\manimce-0.1.1.post2.dist-info\METADATA
    c:\python39\lib\site-packages\manim\manimce-0.1.1.post2.dist-info\RECORD
    c:\python39\lib\site-packages\manim\manimce-0.1.1.post2.dist-info\WHEEL
    c:\python39\lib\site-packages\manim\manimce-0.1.1.post2.dist-info\entry_points.txt
Proceed (Y/n)? Y
  Successfully uninstalled manimce-0.1.1.post2

C:\Users\Rene>manim
'manim' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Rene>pip list
Package            Version
------------------ ---------
backcall           0.2.0
cairocffi          1.2.0
certifi            2021.10.8
cffi               1.14.6
charset-normalizer 2.0.7
click              8.0.3
colorama           0.4.4
colour             0.1.5
commonmark         0.9.1
cycler             0.10.0
ffmpeg             1.4
glcontext          2.3.4
idna               3.3
isosurfaces        0.1.0
jedi               0.18.0
kiwisolver         1.3.2
matplotlib-inline  0.1.3
moderngl           5.6.4
mpmath             1.2.1
multipledispatch   0.6.0
networkx           2.6.3
numpy              1.21.3
odeintw            0.1.1
opencv-python      4.5.3.56
pangocairocffi     0.4.0
pangocffi          0.8.0
parso              0.8.2
pickleshare        0.7.5
Pillow             8.4.0
pip                21.2.3
progressbar        2.5
prompt-toolkit     3.0.20
pycairo            1.20.1
pycparser          2.20
pydub              0.25.1
pyglet             1.5.21
Pygments           2.10.0
pyparsing          2.4.7
pyreadline         2.1
pyrr               0.10.3
python-dateutil    2.8.2
requests           2.26.0
rich               6.2.0
scipy              1.7.1
screeninfo         0.6.7
setuptools         57.4.0
six                1.16.0
skia-pathops       0.7.0
sympy              1.9
tqdm               4.62.3
traitlets          5.1.0
typing-extensions  3.10.0.2
urllib3            1.26.7
validators         0.18.2
watchdog           2.1.6
wcwidth            0.2.5
wheel              0.37.0

C:\Users\Rene>pip install manim
Collecting manim
  Using cached manim-0.12.0-py3-none-any.whl (505 kB)
Requirement already satisfied: numpy<2.0,>=1.9 in c:\python39\lib\site-packages (from manim) (1.21.3)
Requirement already satisfied: click>=7.1 in c:\python39\lib\site-packages (from manim) (8.0.3)
Requirement already satisfied: pydub in c:\python39\lib\site-packages (from manim) (0.25.1)
Requirement already satisfied: setuptools in c:\python39\lib\site-packages (from manim) (57.4.0)
Requirement already satisfied: colour in c:\python39\lib\site-packages (from manim) (0.1.5)
Collecting moderngl-window<3.0.0,>=2.3.0
  Using cached moderngl_window-2.4.0-py3-none-any.whl (362 kB)
Collecting decorator<6.0.0,>=5.0.7
  Using cached decorator-5.1.0-py3-none-any.whl (9.1 kB)
Requirement already satisfied: screeninfo<0.7.0,>=0.6.7 in c:\python39\lib\site-packages (from manim) (0.6.7)
Requirement already satisfied: isosurfaces==0.1.0 in c:\python39\lib\site-packages (from manim) (0.1.0)
Requirement already satisfied: networkx<3.0,>=2.5 in c:\python39\lib\site-packages (from manim) (2.6.3)
Collecting cloup<0.8.0,>=0.7.0
  Using cached cloup-0.7.1-py2.py3-none-any.whl (23 kB)
Requirement already satisfied: requests in c:\python39\lib\site-packages (from manim) (2.26.0)
Collecting manimpango<0.4.0,>=0.3.0
  Using cached ManimPango-0.3.1-cp39-cp39-win_amd64.whl (3.9 MB)
Collecting click-default-group
  Using cached click_default_group-1.2.2-py3-none-any.whl
Requirement already satisfied: skia-pathops<0.8.0,>=0.7.0 in c:\python39\lib\site-packages (from manim) (0.7.0)
Requirement already satisfied: rich>=6.0 in c:\python39\lib\site-packages (from manim) (6.2.0)
Requirement already satisfied: pygments in c:\python39\lib\site-packages (from manim) (2.10.0)
Requirement already satisfied: moderngl<6.0.0,>=5.6.3 in c:\python39\lib\site-packages (from manim) (5.6.4)
Requirement already satisfied: Pillow in c:\python39\lib\site-packages (from manim) (8.4.0)
Requirement already satisfied: tqdm in c:\python39\lib\site-packages (from manim) (4.62.3)
Requirement already satisfied: scipy in c:\python39\lib\site-packages (from manim) (1.7.1)
Collecting mapbox-earcut<0.13.0,>=0.12.10
  Using cached mapbox_earcut-0.12.11-cp39-cp39-win_amd64.whl (67 kB)
Requirement already satisfied: watchdog in c:\python39\lib\site-packages (from manim) (2.1.6)
Requirement already satisfied: pycairo<2.0,>=1.19 in c:\python39\lib\site-packages (from manim) (1.20.1)
Requirement already satisfied: colorama in c:\python39\lib\site-packages (from click>=7.1->manim) (0.4.4)
Requirement already satisfied: glcontext<3,>=2 in c:\python39\lib\site-packages (from moderngl<6.0.0,>=5.6.3->manim) (2.3.4)
Requirement already satisfied: pyrr<1,>=0.10.3 in c:\python39\lib\site-packages (from moderngl-window<3.0.0,>=2.3.0->manim) (0.10.3)
Requirement already satisfied: pyglet<2,>=1.5.8 in c:\python39\lib\site-packages (from moderngl-window<3.0.0,>=2.3.0->manim) (1.5.21)
Requirement already satisfied: multipledispatch in c:\python39\lib\site-packages (from pyrr<1,>=0.10.3->moderngl-window<3.0.0,>=2.3.0->manim) (0.6.0)
Requirement already satisfied: typing-extensions<4.0.0,>=3.7.4 in c:\python39\lib\site-packages (from rich>=6.0->manim) (3.10.0.2)
Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in c:\python39\lib\site-packages (from rich>=6.0->manim) (0.9.1)
Requirement already satisfied: six in c:\python39\lib\site-packages (from multipledispatch->pyrr<1,>=0.10.3->moderngl-window<3.0.0,>=2.3.0->manim) (1.16.0)
Requirement already satisfied: idna<4,>=2.5 in c:\python39\lib\site-packages (from requests->manim) (3.3)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in c:\python39\lib\site-packages (from requests->manim) (1.26.7)
Requirement already satisfied: charset-normalizer~=2.0.0 in c:\python39\lib\site-packages (from requests->manim) (2.0.7)
Requirement already satisfied: certifi>=2017.4.17 in c:\python39\lib\site-packages (from requests->manim) (2021.10.8)
Installing collected packages: moderngl-window, mapbox-earcut, manimpango, decorator, cloup, click-default-group, manim
  WARNING: Failed to write executable - trying to use .deleteme logic
ERROR: Could not install packages due to an OSError: [WinError 2] The system cannot find the file specified: 'C:\\Python39\\Scripts\\manim.exe' -> 'C:\\Python39\\Scripts\\manim.exe.deleteme'


C:\Users\Rene>pip uninstall manimce --user

Usage:
  pip uninstall [options] <package> ...
  pip uninstall [options] -r <requirements file> ...

no such option: --user

C:\Users\Rene>pip uninstall manimce --Rene

Usage:
  pip uninstall [options] <package> ...
  pip uninstall [options] -r <requirements file> ...

no such option: --Rene

C:\Users\Rene>
`

**Reply by hackyfemmyboi** [reply]
heres what i did in the cmd

**Reply by Darylgolden** [reply]
Try `pip install --user manim`

**Reply by hackyfemmyboi** [reply]
`Microsoft Windows [Version 10.0.18363.418]
(c) 2019 Microsoft Corporation. All rights reserved.

C:\Users\Rene>pip install --user manim
Requirement already satisfied: manim in c:\python39\lib\site-packages (0.12.0)
Requirement already satisfied: networkx<3.0,>=2.5 in c:\python39\lib\site-packages (from manim) (2.6.3)
Requirement already satisfied: moderngl-window<3.0.0,>=2.3.0 in c:\python39\lib\site-packages (from manim) (2.4.0)
Requirement already satisfied: colour in c:\python39\lib\site-packages (from manim) (0.1.5)
Requirement already satisfied: skia-pathops<0.8.0,>=0.7.0 in c:\python39\lib\site-packages (from manim) (0.7.0)
Requirement already satisfied: manimpango<0.4.0,>=0.3.0 in c:\python39\lib\site-packages (from manim) (0.3.1)
Requirement already satisfied: requests in c:\python39\lib\site-packages (from manim) (2.26.0)
Requirement already satisfied: Pillow in c:\python39\lib\site-packages (from manim) (8.4.0)
Requirement already satisfied: numpy<2.0,>=1.9 in c:\python39\lib\site-packages (from manim) (1.21.3)
Requirement already satisfied: decorator<6.0.0,>=5.0.7 in c:\python39\lib\site-packages (from manim) (5.1.0)
Requirement already satisfied: pygments in c:\python39\lib\site-packages (from manim) (2.10.0)
Requirement already satisfied: rich>=6.0 in c:\python39\lib\site-packages (from manim) (6.2.0)
Requirement already satisfied: tqdm in c:\python39\lib\site-packages (from manim) (4.62.3)
Requirement already satisfied: isosurfaces==0.1.0 in c:\python39\lib\site-packages (from manim) (0.1.0)
Requirement already satisfied: moderngl<6.0.0,>=5.6.3 in c:\python39\lib\site-packages (from manim) (5.6.4)
Requirement already satisfied: cloup<0.8.0,>=0.7.0 in c:\python39\lib\site-packages (from manim) (0.7.1)
Requirement already satisfied: pydub in c:\python39\lib\site-packages (from manim) (0.25.1)
Requirement already satisfied: scipy in c:\python39\lib\site-packages (from manim) (1.7.1)
Requirement already satisfied: screeninfo<0.7.0,>=0.6.7 in c:\python39\lib\site-packages (from manim) (0.6.7)
Requirement already satisfied: pycairo<2.0,>=1.19 in c:\python39\lib\site-packages (from manim) (1.20.1)
Requirement already satisfied: watchdog in c:\python39\lib\site-packages (from manim) (2.1.6)
Requirement already satisfied: click>=7.1 in c:\python39\lib\site-packages (from manim) (8.0.3)
Requirement already satisfied: setuptools in c:\python39\lib\site-packages (from manim) (57.4.0)
Requirement already satisfied: mapbox-earcut<0.13.0,>=0.12.10 in c:\python39\lib\site-packages (from manim) (0.12.11)
Requirement already satisfied: click-default-group in c:\python39\lib\site-packages (from manim) (1.2.2)
Requirement already satisfied: colorama in c:\python39\lib\site-packages (from click>=7.1->manim) (0.4.4)
Requirement already satisfied: glcontext<3,>=2 in c:\python39\lib\site-packages (from moderngl<6.0.0,>=5.6.3->manim) (2.3.4)
Requirement already satisfied: pyrr<1,>=0.10.3 in c:\python39\lib\site-packages (from moderngl-window<3.0.0,>=2.3.0->manim) (0.10.3)
Requirement already satisfied: pyglet<2,>=1.5.8 in c:\python39\lib\site-packages (from moderngl-window<3.0.0,>=2.3.0->manim) (1.5.21)
Requirement already satisfied: multipledispatch in c:\python39\lib\site-packages (from pyrr<1,>=0.10.3->moderngl-window<3.0.0,>=2.3.0->manim) (0.6.0)
Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in c:\python39\lib\site-packages (from rich>=6.0->manim) (0.9.1)
Requirement already satisfied: typing-extensions<4.0.0,>=3.7.4 in c:\python39\lib\site-packages (from rich>=6.0->manim) (3.10.0.2)
Requirement already satisfied: six in c:\python39\lib\site-packages (from multipledispatch->pyrr<1,>=0.10.3->moderngl-window<3.0.0,>=2.3.0->manim) (1.16.0)
Requirement already satisfied: certifi>=2017.4.17 in c:\python39\lib\site-packages (from requests->manim) (2021.10.8)
Requirement already satisfied: charset-normalizer~=2.0.0 in c:\python39\lib\site-packages (from requests->manim) (2.0.7)
Requirement already satisfied: idna<4,>=2.5 in c:\python39\lib\site-packages (from requests->manim) (3.3)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in c:\python39\lib\site-packages (from requests->manim) (1.26.7)

C:\Users\Rene>manim
'manim' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Rene>`

**Reply by hackyfemmyboi** [reply]
didnt work

**Reply by Darylgolden** [reply]
Now try `python -m manim`.

**Reply by hackyfemmyboi** [reply]
ahhhhh there we go!

**Reply by hackyfemmyboi** [reply]
thanks sooo much Darylgolden

**Reply by hackyfemmyboi** [reply]
Also, how do i animate pi creatures

**Comment by hackyfemmyboi** [comment]
absolutely. I tried everything
